src/io.rs

```diff
@@ -8,27 +9,24 @@ use std::{
     io::{self, Read, Write},
 };
 
-/// process each filepath in the input Vec, or stdin if empty
-pub fn process_files(filepaths: Vec<String>) -> Result<()> {
+/// process first filepath in the input Vec, or stdin if empty
```

Comment:
Should we fail if there are multiple files?

Reply:
yes, coming in a future PR that hooks up `structopt`